src/trumbowyg.js

```diff
@@ -1232,6 +1248,15 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                 return true;
             });
         },
+        prependHttpProtocol: function(url) {
+            var t = this;
+            if(!t.o.autoPrefixHttpProtocol) { return url; }
+
+            const VALID_LINK_PREFIX = /^([a-z][-+.a-z0-9]*:|\/|#)/i;
+            if(VALID_LINK_PREFIX.test(url)) { return url; }
+
+            return url.includes("@") ? `mailto:${url}` : `http://${url}`;
```

Comment:
That's wrong, you can use @ in HTTP url to put some user/password:

`[//[user[:password]@]host[:port]][/path][?query][#fragment]`